lib/geolocation/index.js

```diff
@@ -73,7 +95,7 @@ module.exports = Geolocation;
 if (module.parent === null) {
   var sensor = new Geolocation();
   sensor.onreading = function() {
-    console.log('log: ' + this.hasReading)
+    console.log('lat:', this.latitude, 'long:', this.longitude, ' - ', this.timestamp)
```

Comment:
; missing here

didnt eslint complain about this?

Reply:
No, it did not. Fixed now, though.